src/make/cards.js

```diff
@@ -292,11 +295,15 @@ function doCard(rawCard, cards, code, set) {
   var color = !colors ? 'colorless' :
     colors.length > 1 ? 'multicolor' :
     colors[0].toLowerCase()
+    
+  //set lands to .1 to sort them before nonland 0cmc  
+  if ('Land'.indexOf(rawCard.types) > -1)
```

Comment:
Shouldn't this be `rawCard.types.indexOf('Land')`, or am I missing something? (Also, will the text `Land` be capitalized in the string? It might be prudent to just lowercase both for safety.)